Dockerfile

```diff
@@ -50,9 +58,11 @@ RUN ARCH="$(uname -m)" && \
 && BIN_LOCATION=$(tar -tzf /tmp/geoipupdate.tgz | grep "/geoipupdate$") \
 && export BIN_LOCATION \
 && tar -xvzf /tmp/geoipupdate.tgz --strip-components=1 --directory=/usr/bin/ "$BIN_LOCATION" \
-&& chown root:root /usr/bin/geoipupdate \
+&& chown "${user}:${user}" /usr/bin/geoipupdate \
```

Comment:
```suggestion
```

Binaries in `/usr/bin` should be owned by `root` (which is already the case once you've downloaded it as the default container user is... `root` until the instruction `USER` is set)